cpp/src/dataframe/cudf_table.cuh

```diff
@@ -614,6 +619,7 @@ public:
    * @Param row_index The row of the table to compute the hash value for
    * @Param num_columns_to_hash The number of columns in the row to hash. If 0, 
    * hashes all columns
+   * @Param initial_hash_values Optional initial hash values to combine with each column's hashed values
```

Comment:
Would appreciate it if you could update this documentation here per: https://github.com/rapidsai/cudf/blob/branch-0.6/cpp/src/examples/documentation_example.cpp

Reply:
Done